packages/react-components/src/trace-explorer/trace-explorer-views-widget.tsx

```diff
@@ -179,13 +179,7 @@ export class ReactAvailableViewsWidget extends React.Component<ReactAvailableVie
         /**
          * Remove outputs of type DATA_TREE since the view is not supported in theia-traceviewer
          */
-        let outputs = outputDescriptors.filter(value => (value.type !== 'DATA_TREE'));
-        /*
-         * Remove certain scatter specific outputs since they are not supported in the theia-traceviewer
-         * There is no generic way to identify such outputs. This is not ideal because other server
-         * implementations are not covered.
-         */
-        outputs = outputs.filter(value => !value.id.includes('scatter'));
+        const outputs = outputDescriptors.filter(value => (value.type !== 'DATA_TREE'));
```

Comment:
Agreed. Disabled.

Reply:
Yeah it can be enabled once users can use it as they would expect (ex. Values can be read, basic navigation features are consistent with the other charts). Thank you for thinking of this! It keeps the tool feeling polished even during development.